Resources/Prototypes/Entities/Mobs/Player/silicon.yml

```
@@ -619,7 +619,8 @@
   - type: BlockMovement
     blockInteraction: false
   - type: SiliconLawProvider
-    laws: Asimov
+    laws: NTDefault # Goobstation - AI/borg law changes - set AI to NTDefault
```

```suggestion
    laws: Augustine
```
I have *repeatedly* said this.

---

Augustine also leaves a huge loophole for Station AI players to basically go "you arent intelligent" and kill people